dashboard/config/scripts/allthethings.script

```diff
@@ -116,8 +116,8 @@ stage 'Studio'
 level 'Resizing Test'
 
 stage 'Long assessment'
-assessment 'Test longassessment levelgroup'
-assessment 'Test longassessment levelgroup split'
+assessment 'Test longassessment levelgroup new'
+assessment 'Test longassessment levelgroup split new'
```

Comment:
We have so few of us using these so far (me, Poorva, you?) that I figured that wouldn't be a big deal.  And we could just create a fresh account on whatever environment we were testing these on, which is likely staging anyway.